com.unity.netcode.gameobjects/Editor/CodeGen/CodeGenHelpers.cs

```diff
@@ -6,39 +6,140 @@
 using Mono.Cecil;
 using Mono.Cecil.Cil;
 using Mono.Cecil.Rocks;
-using Unity.Collections;
 using Unity.CompilationPipeline.Common.Diagnostics;
 using Unity.CompilationPipeline.Common.ILPostProcessing;
-using UnityEngine;
+using Unity.Netcode.Shared;
 
 namespace Unity.Netcode.Editor.CodeGen
 {
+    namespace Import
+    {
+#pragma warning disable IDE1006 // disable naming rule violation check
+        internal static class ModuleNames
+        {
+            public const string NetcodeRuntime = "Unity.Netcode.Runtime.dll";
+            public const string NetStandard = "netstandard.dll";
+            public const string UnityEngineCoreModule = "UnityEngine.CoreModule.dll";
+        }
+
+        internal static class TypeNames
+        {
+            public const string NetworkManager = "Unity.Netcode.NetworkManager";
```

Comment:
rather than hardcoding all string by hand here, we could still reference Netcode.Runtime's types but instead of `typeof(XXX)` type-based lookup in the ILPP/Cecil code, we could simply produce strings here with `nameof(XXX)` to keep it consistent with the Netcode.Runtime — also by doing that, we keep Netcode.Runtime reference around and we don't need to have `Unity.Netcode.Shared` asmdef and/or duplicate `xxHash` implementation, we could simply revert them back to current state.